### DISCUSSION TITLE
Module build failed: Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.0". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.

### DISCUSSION THREAD

**Discussion by Er-Ramanand-Dubey** [babel/babel#12009] [Q&A]
Can anyone help me out? when I run the command **yarn run build**. I am getting this error, I don't know what is this error and how to resolve this.

ERROR  Failed to compile with 1 errors                                                                                                                       9:43:18 PM

 error  in ./client/polyfills.js

Module build failed: Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.0". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.
    at throwVersionError (C:\Users\MY HP\Desktop\project updating\version update\node_modules\@babel\helper-plugin-utils\lib\index.js:65:11)
    at assertVersion (C:\Users\MY HP\Desktop\project updating\version update\node_modules\@babel\helper-plugin-utils\lib\index.js:13:11)
    at C:\Users\MY HP\Desktop\project updating\version update\node_modules\babel-plugin-inline-react-svg\lib\index.js:43:3
    at C:\Users\MY HP\Desktop\project updating\version update\node_modules\@babel\helper-plugin-utils\lib\index.js:19:12
    at Function.memoisePluginContainer (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-core\lib\transformation\file\options\option-manager.js:113:13)
    at Function.normalisePlugin (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-core\lib\transformation\file\options\option-manager.js:146:32)
    at C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-core\lib\transformation\file\options\option-manager.js:184:30
    at Array.map (<anonymous>)
    at Function.normalisePlugins (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-core\lib\transformation\file\options\option-manager.js:158:20)
    at OptionManager.mergeOptions (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-core\lib\transformation\file\options\option-manager.js:234:36)
    at OptionManager.init (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-core\lib\transformation\file\options\option-manager.js:368:12)
    at File.initOptions (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-core\lib\transformation\file\index.js:212:65)
    at new File (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-core\lib\transformation\file\index.js:135:24)
    at Pipeline.transform (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-core\lib\transformation\pipeline.js:46:16)
    at transpile (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-loader\lib\index.js:50:20)
    at C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-loader\lib\fs-cache.js:118:18
    at ReadFileContext.callback (C:\Users\MY HP\Desktop\project updating\version update\node_modules\next\node_modules\babel-loader\lib\fs-cache.js:31:21)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:260:13)

 @ multi ./client/polyfills.js ./node_modules/next/dist/client/webpack-hot-middleware-client ./node_modules/next/dist/client/on-demand-entries-client ./node_modules/next/dist/client/next-dev.js

(node:10024) DeprecationWarning: Module.chunks: Use Module.forEachChunk/mapChunks/getNumberOfChunks/isInChunk/addChunk/removeChunk instead
> Ready on http://localhost:3000

**my package.json look like this**.

```
{
  "name": "nextteo",
  "version": "0.0.1",
  "license": "MIT",
  "scripts": {
    "dev": "next",
    "build": "next build",
    "start": "NODE_ENV=production node server"
  },
  "now": {
    "alias": "next.teo.systems",
    "env": {
      "PORT": "8080"
    }
  },
  "engines": {
    "node": "^13.9.0"
  },
  "dependencies": {
    "@babel/plugin-transform-runtime": "^7.11.0",
    "@sentry/node": "^5.4.0",
    "axios": "^0.16.2",
    "axs": "^1.0.6",
    "babel-plugin-transform-class-properties": "^6.24.1",
    "babel-plugin-transform-object-rest-spread": "^6.26.0",
    "babel-polyfill": "^6.26.0",
    "babel-preset-next-react": "^1.1.1",
    "babel-runtime": "^6.26.0",
    "browser-history": "^1.0.1",
    "chart": "^0.1.2",
    "compression-webpack-plugin": "^5.0.1",
    "core-js": "^3.6.5",
    "css": "^2.2.1",
    "express": "^4.16.3",
    "express-force-https": "^1.0.0",
    "express-https-redirect": "^1.0.0",
    "isomorphic-fetch": "^2.2.1",
    "jquery": "^3.2.1",
    "js-cookie": "^2.2.1",
    "lodash": "^4.17.4",
    "lru-cache": "^4.1.2",
    "moment": "^2.21.0",
    "moment-timezone": "^0.5.14",
    "next": "^4.2.3",
    "next-redux-wrapper": "^1.1.3",
    "next-routes": "^1.2.0",
    "nodemailer": "^4.6.3",
    "nodemon": "^1.19.1",
    "pm2": "^3.5.1",
    "promise-polyfill": "8.1.0",
    "prop-types": "^15.6.1",
    "query-string": "^5.1.1",
    "react": "^16.8.4",
    "react-autosuggest": "^9.3.0",
    "react-bootstrap": "^0.31.2",
    "react-bootstrap-dialog": "^0.9.4",
    "react-dom": "^16.8.4",
    "react-dropzone": "^4.2.9",
    "react-ga": "^2.5.6",
    "react-helmet": "^5.2.0",
    "react-icons": "^2.2.5",
    "react-image-gallery": "^0.8.3",
    "react-image-lightbox": "^5.1.0",
    "react-img-zoom": "^0.1.0",
    "react-imgix": "^7.1.1",
    "react-instantsearch": "^4.5.2",
    "react-mailto": "^0.4.0",
    "react-native": "^0.63.2",
    "react-native-network-info": "^5.2.1",
    "react-owl-carousel": "^2.3.1",
    "react-recaptcha": "^2.3.10",
    "react-redux": "^5.0.5",
    "react-router-dom": "^4.2.2",
    "react-sanfona": "^1.0.6",
    "react-slick": "0.16.0",
    "react-test-renderer": "^16.8.4",
    "redux": "^3.7.0",
    "redux-actions": "^2.3.0",
    "redux-responsive": "^4.3.6",
    "redux-thunk": "^2.2.0",
    "reline": "^1.0.0-beta.3",
    "sitemap": "^2.1.0",
    "soap": "^0.20.0",
    "stylis": "^3.0.0",
    "uglifyjs-webpack-plugin": "^2.2.0",
    "webpack": "^4.0.0",
    "webpack-dev-middleware": "^3.7.2",
    "webpack-dev-server": "^3.11.0",
    "yarn": "^1.6.0",
    "zone.js": "^0.8.20"
  },
  "devDependencies": {
    
    "babel-core": "^7.0.0-bridge.0",
    "babel-polyfill": "^6.26.0",
    "babel-eslint": "^10.1.0",
    "babel-plugin-inline-react-svg": "^1.1.1",
    "babel-plugin-static-fs": "^1.1.0",
    "eslint": "^7.7.0",
    "eslint-config-xo-react": "^0.12.0",
    "eslint-plugin-react": "^7.20.6",
    "sw-precache-webpack-plugin": "^1.0.0",
    "xo": "^0.18.1"
  },
  "resolutions": {
    "**/**/lodash": "^4.17.12"
  },
  "xo": {
    "envs": [
      "node",
      "es6",
      "browser"
    ],
    "parser": "babel-eslint",
    "extends": "xo-react",
    "space": true,
    "rules": {
      "unicorn/number-literal-case": 0,
      "no-multi-assign": 1,
      "array-bracket-spacing": [
        2,
        "always"
      ],
      "object-curly-spacing": [
        "error",
        "always"
      ],
      "comma-dangle": [
        "error",
        "always-multiline"
      ],
      "arrow-parens": [
        2,
        "as-needed",
        {
          "requireForBlockBody": true
        }
      ],
      "operator-linebreak": 0,
      "capitalized-comments": 0,
      "react/no-danger": 1,
      "react/no-array-index-key": 0,
      "react/jsx-closing-bracket-location": [
        2,
        "tag-aligned"
      ],
      "react/jsx-space-before-closing": [
        2,
        "always"
      ],
      "react/jsx-tag-spacing": [
        2,
        {
          "beforeSelfClosing": "always"
        }
      ],
      "react/jsx-no-bind": 0,
      "react/forbid-component-props": 0,
      "react/require-default-props": 0,
      "react/jsx-handler-names": 0
    }
  }
}
```
**my .bablerc file looks like this**

```
{
  "presets": [ "next/babel" ],
    "plugins": [ "inline-react-svg","transform-object-rest-spread",
    "transform-class-properties",
    "transform-runtime","babel-plugin-static-fs"]
}
```
**My next.config.js look like**

```
const path = require('path');
const SWPrecacheWebpackPlugin = require('sw-precache-webpack-plugin');
var CompressionPlugin = require("compression-webpack-plugin");
var UglifyJsPlugin = require('uglifyjs-webpack-plugin')
var webpack = require('webpack');
module.exports = {
	    plugins: [

		    			new webpack.DefinePlugin({
									  'process.env': {
										  					'NODE_ENV': JSON.stringify('production')
										  				}
									}),

		    			new UglifyJsPlugin({
										uglifyOptions: {
																output: {comments: false,beautify: false},
																compress: true,
																warnings: false
															}
									}),
		    			new CompressionPlugin({
							          filename: "[path].gz[query]",
									  algorithm: "gzip",
									  test: /\.js$|\.css$|\.html$/,
									  threshold: 10240,
									  minRatio: 0.8
									}),

		    			new SWPrecacheWebpackPlugin({
									  cacheId: 'next-ss',
									  filename: 'sw.js',
									  minify: true,
									  staticFileGlobsIgnorePatterns: [ /\.next\// ],
									  staticFileGlobs: [
										  				'static/**/*',
										  			  ],
									  runtimeCaching: [
										  				{
																			  handler: 'networkFirst',
																			  urlPattern: /^https?.*/,
																			},
										  			  ],
									})

		       ],
	  	webpack: function (cfg) {
			    const originalEntry = cfg.entry
			    cfg.entry = async () => {
				          const entries = await originalEntry()
						  if (
							entries['main.js'] &&
							!entries['main.js'].includes('./client/polyfills.js')
						  ) {
							entries['main.js'].unshift('./client/polyfills.js')
						  }
				          return entries
				        }

			    return cfg
				}
};
```

**Comment by JLHwung** [comment]
Please uninstall `babel-*` and install `@babel/*`. i.e. `babel-core` to `@babel/core`: See also the migration guide from v6 to v7: https://babel.dev/docs/en/v7-migration#scoped-packages
